src/Mod/Draft/drafttaskpanels/task_scale.py

```diff
@@ -46,28 +46,30 @@ class ScaleTaskPanel:
     """The task panel for the Draft Scale tool."""
 
     def __init__(self):
+        decimals = App.ParamGet("User parameter:BaseApp/Preferences/Units").GetInt("Decimals", 2)
         self.sourceCmd = None
         self.form = QtGui.QWidget()
+        self.form.setWindowIcon(QtGui.QIcon(":/icons/Draft_Scale.svg"))
         layout = QtGui.QGridLayout(self.form)
         self.xLabel = QtGui.QLabel()
         layout.addWidget(self.xLabel, 0, 0, 1, 1)
         self.xValue = QtGui.QDoubleSpinBox()
         self.xValue.setRange(0.0000001, 1000000.0)
-        self.xValue.setDecimals(Draft.precision())
+        self.xValue.setDecimals(decimals)
```

Comment:
value from `GetInt()` is assigned as a decimal?

Reply:
No, this is the number of decimals.